common/src/test/kotlin/com/hartwig/actin/molecular/evidence/EvidenceDatabaseTest.kt

```diff
@@ -28,32 +28,32 @@ class EvidenceDatabaseTest {
             driverLikelihood = DriverLikelihood.HIGH,
             isReportable = true
         )
-        assertThat(database.geneAlterationForVariant(variant)).isNotNull()
+        assertThat(database.alterationForVariant(variant)).isNotNull()
```

Comment:
This test (and similar tests in this class) are meaningless now that evidence database always returns not-null anyways